components/ComparisonsDisplay.vue

```diff
@@ -50,7 +50,14 @@ export default class ComparisonsDisplay extends Vue {
   cumulateTime: boolean = true;
 
   subsplitLabel(name: string) {
-    return name.startsWith('-') ? `${name.substring(1)} (subsplit)` : name;
+    if (name.startsWith('-')) {
```

Comment:
This small if / else if / else is duplicated several times, it should probably be only once somewhere in the `util` folder 

(also we don't need an if, an else if and an else, because each breach has a return, you can just do
```js
if () {
   return ...;
}

if () {
   return ...;
}

return ...;